scripts/build/wrap_binary_in_venv.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python2
+"""
+Create a wrapper for a runnable file which requires a certain virtualenv to
+provide its dependencies. The wrapper implicitly uses the virtual environment,
+so it does not need to activated explicitly.
+"""
+
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+from logging import getLogger, Formatter, StreamHandler, INFO, DEBUG
+from os import stat, chmod
+from os.path import abspath
+from stat import S_IXUSR
+from string import Template
+
+LOG = getLogger('VirtualenvWrapper')
+
+msg_formatter = Formatter('[%(levelname)s] - %(message)s')
+log_handler = StreamHandler()
+log_handler.setFormatter(msg_formatter)
+LOG.setLevel(INFO)
+LOG.addHandler(log_handler)
+
+
+def add_executable_permission(path):
+    current_stat = stat(path)
+    chmod(path, current_stat.st_mode | S_IXUSR)
+
+
+def generate_content(binary_path, output_path, virtual_environment_path):
+
+    template_content = '\n'.join([
+        r'#!/bin/bash',
+        r'source "${virtual_environment_path}/bin/activate"',
+        r'exec "${binary_path}" "$$@"'])
+
+    template_vars = {
+        'binary_path': binary_path,
+        'output_path': output_path,
```

Comment:
Where is the `output_path` used in the template?

Reply:
Thanks for catching this, it was never used. Fixed.